sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -47,11 +47,14 @@ export interface Receiver<ReceivedMessageT> {
   getMessageIterator(options?: GetMessageIteratorOptions): AsyncIterableIterator<ReceivedMessageT>;
 
   /**
-   * Receives, at most, `maxMessages` worth of messages.
-   * @param maxMessages The maximum number of messages to accept.
+   * Receives, at most, `maxMessageCount` worth of messages.
```

Comment:
```suggestion
   * Receives, at most, `maxMessageCount` number of messages.
```
??

Reply:
I have a separate PR fixing up the docs as there are quite a few to be fixed :)